test/e2e-png.js

```diff
@@ -43,6 +48,18 @@ test.beforeEach(async t => {
     images = [];
 });
 
+test("Remove original file if move option is supplied", async t => {
+    const spy = sinon.spy(fileSystem, "deleteObject");
+    const im = await processor.run(new Config({
+        "bucket": "some",
+        "backup": {
+            "move": true
+        }
+    }));
+    t.true(spy.called);
+    images = [];
```

Comment:
I think these testing sometimes fails due to asynchronous processing.
These are better to assert image count from `processor.run` result.
But in this case we cannot assert processed images...